manifests/Nekres/Quick_Surrender/0.1.7.json

```diff
@@ -1,19 +1,19 @@
 {
   "manifest_version": "1",
   "name": "Quick Surrender",
-  "namespace": "Nekres.Quick_Surrender_Module",
-  "version": "0.1.6",
+  "namespace": "Nekres.Quick_Surrender",
```

Comment:
Why change the namespace?  This will break the upgrade path and any configured settings for anyone using your module.  Can this not be kept as is?  This will also mean every user will have two Quick Surrender modules if they try to add the new one and already had the old one.

Reply:
I want to have every namespace use it's name without the Module after it.    I noticed that I did the same for Region Of Tyria so I wanted to standardize the rest.
They'll have to deal with it once. I mean bhud is still 0.11 and the library is rather new so I guess everyone knows where the module folder is.

Reply:
Quick Surrender has nearly 2 thousand downloads, so it's not worth it.  There's no reason to change the namespace.  It doesn't affect anything visually.

It will only hurt the user experience.  We made that mistake already and don't want to do it again with any other modules.

Reply:
Closing this. Namespace is reverted in #67